src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/UkfModel.h

```diff
@@ -89,7 +89,7 @@ class UkfModel
     std::vector<Eigen::MatrixXd> m_tempJacobianList; /**< List of jacobians per eache submodel. */
     manif::SE3d m_worldTBase; /**< Sub-model base pose wrt the inertial frame */
     int m_offsetMeasurement; /**< Offset used to fill the measurement vector. */
-    std::map<std::string, std::string> m_stateToUkfNames; /**< Map used to retrieve the name of the variable passed as state and the ukf name. */
+    std::map<std::pair<std::string, std::string>, std::string> m_variableNameToUkfState; /**< Map used to retrieve the name of the variable passed as state and the ukf name. */
```

Comment:
Can you use a struct? Or at least improve the documentation explaining the meaning of the keys and values

Reply:
Done